content/docs/command-reference/remote/modify.md

```diff
@@ -333,14 +333,21 @@ following parameters (listed in order or precedence) to customize the
 authentication method:
 
 1. `connection_string` is used for authentication if given (all others params
-   are ignored).
+   are ignored). If `connection_string` is not set, the `account_name` must be
+   given.
 2. If `tenant_id` and `client_id` or `client_secret` are given, Active Directory
    (AD)
    [service principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
    auth is performed.
-3. The storage `account_name` is tried next, along with `account_key` or
-   `sas_token` (in that order). If neither key nor token is provided, DVC will
-   try to connect anonymously.
+3. If either `account_key` or `sas_token` (in this order) are set, then DVC will
+   try to connect with those options.
+4. If `allow_anonymous_login` is set to `True`, then DVC will try to connect
+   anonymously.
+5. If none of the steps apply, and the only variable which is set is
+   `account_name` then DVC will try to use a
```

Comment:
```suggestion
5. If none of the steps above apply, and the only variable that is set is
   `account_name`, then DVC will try to use a
```

I think @jorgeorpinel would prefer "that."

Reply:
Or neither ("the only variable set"). Wouldn't call these steps though. Updating and committing along with any other copy edits...